general-concepts/ebuild-revisions/text.xml

```diff
@@ -54,13 +54,48 @@ of thumb could be used as a guideline:
   </li>
 </ol>
 
-<p>
-Simple compile fixes do <b>not</b> warrant a revision bump; this is because they do
-not affect the installed package for users who already managed to compile it.
-Small documentation fixes are also usually not grounds for a new revision.
-In particular, this applies if the package has a substantial compilation
-time; developers should use their best judgement in these circumstances.
-</p>
+<p>The examples of changes that warrant a new revision are:</p>
+<ul>
+  <li>adding a patch to fix a runtime issue,</li>
+  <li>installing additional files that could be useful to the user,</li>
+  <li>adding a missing runtime dependency to one of the existing blocks,</li>
```

Comment:
I would add the general rule to do a rev bump whenever you make changes to any installed files or changes the set of installed files (remove/add new files). Then add exceptions (like the trivial documentation fix which maybe changes a man file or program's `-h` output).